deploy/webhook.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+  annotations:
+    certmanager.k8s.io/inject-ca-from: istio-operator/webhook-server-cert
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: istio-operator
+        namespace: istio-operator
+        path: /validate-install-istio-io-v1alpha2-istiocontrolplane
```

Comment:
Is this correct? since you have register validate path `/validate` in webhook server.

Reply:
because I have done multiple trials in webhook server with different combination, actually I tried using the /validate path here or change the path during registration but seems no luck

Reply:
As I mentioned previously, the webhook defined in the yaml is irrelevant.  All of this gets defined when the webhook is configured in the code.

Reply:
for controller-runtime v2 we have to create this configuration to replace path of previous registration